src/manifest.c

```diff
@@ -729,7 +729,7 @@ struct manifest *load_manifest(int version, struct file *file, struct manifest *
 			retried = true;
 			goto retry_load;
 		}
-		fprintf(stderr, "Failed to load %d %s manifest\n", version, file->filename);
+		info("Failed to load %d %s manifest\n", version, file->filename);
```

Comment:
"failed" messages are probably error() or warn().
Use error() if we are going to abort and warn() if we are going to try to do something else.

Reply:
I didn't do that as part of this PR because of two reasons:
1) that was going to change the swupd output, which was going to break the current tests.
2) we need to consider how we want to handle this, because using the error() function is going to prepend an "Error: " label to the message, but sometimes error messages are cascade from within inner functions, so we might end up having lots of "Error" labels in the output. For example, in this particular case, let's say we are trying to load a manifest with an invalid number of files in it. These are the messages that would be printed now:
```
Error: preposterous (9000000) number of files in test-bundle Manifest, more than 4 million skipping
Warning: Removing corrupt Manifest.test-bundle artifacts and re-downloading...
Error: preposterous (9000000) number of files in test-bundle Manifest, more than 4 million skipping
Failed to load 10 test-bundle manifest
Unable to download manifest test-bundle version 10, exiting now
Failed to install 1 of 1 bundles
```

If we use error() for every error message we are going to end up with something like this:
```
Error: preposterous (9000000) number of files in test-bundle Manifest, more than 4 million skipping
Warning: Removing corrupt Manifest.test-bundle artifacts and re-downloading...
Error: preposterous (9000000) number of files in test-bundle Manifest, more than 4 million skipping
Error: Failed to load 10 test-bundle manifest
Error: Unable to download manifest test-bundle version 10, exiting now
Error: Failed to install 1 of 1 bundles
```

which doesn't look pretty, it would seem like multiple errors happened while it was just one that was printed many times in different parts of the code.

Reply:
@otaviobp what do you think we should do about that ^?

Reply:
I think that all messages that are errors/warnings/fails should use warn() or error() and have the prefix appended. We can review the messages to avoid cascading that much (maybe by using debug() for some).

It's totally ok to let this out of this patch though, because of fixing tests. And fix that later, in another commit.

Reply:
> I think that all messages that are errors/warnings/fails should use warn() or error() and have the prefix appended. We can review the messages to avoid cascading that much (maybe by using debug() for some).

Make sense